js/main.js

```diff
@@ -0,0 +1,26 @@
+//Функция возврата целого числа, взято из MDN
+
+function getWholeNumber(min, max) {
```

Comment:
Не совсем корректный нейминг здесь больше подойдет integer
```suggestion
function getRandomInteger(min, max) {
```